modules/videopress/class.videopress-options.php

```diff
@@ -5,9 +5,6 @@ class VideoPress_Options {
 	/** @var string */
 	public static $option_name = 'videopress';
 
-	/** @var array */
-	public static $jetpack_plans_with_videopress = array( 'jetpack_premium', 'jetpack_business' );
```

Comment:
Noting that this wasn't used anymore, since a714b0b8e568c8bd552e65072bb942c504b43748, so I removed it to avoid confusions.